src/fragments/lib/auth/flutter/getting_started/70_configureBackend.mdx

```diff
@@ -0,0 +1,155 @@
+
+</Block>
+
+<Block name="Manual Configuration">
+
+Existing Authentication resources from AWS (e.g. Amazon Cognito UserPools or Identity Pools) can be used with the Amplify Libraries by calling the `Amplify.configure()` method.
+
+If you are not using the Amplify CLI, a Cognito User Pool and Identity Pool can be used by referencing them in your `amplifyconfiguration.dart` file:
+
+```jsx
+const amplifyconfig = ''' {
+    "UserAgent": "aws-amplify-cli/2.0",
+    "Version": "1.0",
+    "auth": {
+        "plugins": {
+            "awsCognitoAuthPlugin": {
+                "IdentityManager": {
+                    "Default": {}
+                },
+                "CredentialsProvider": {
+                    "CognitoIdentity": {
+                        "Default": {
+                            "PoolId": "[COGNITO IDENTITY POOL ID]",
+                            "Region": "[REGION]"
+                        }
+                    }
+                },
+                "CognitoUserPool": {
+                    "Default": {
+                        "PoolId": "[COGNITO USER POOL ID]",
+                        "AppClientId": "[COGNITO USER POOL APP CLIENT ID]",
+                        "Region": "[REGION]"
+                    }
+                },
+                "Auth": {
+                    "Default": {
+                        "authenticationFlowType": "USER_SRP_AUTH",
+                        "OAuth": {
+                            "WebDomain": "[YOUR COGNITO DOMAIN ]",
+                            "AppClientId": "[COGNITO USER POOL APP CLIENT ID]",
+                            "SignInRedirectURI": "[CUSTOM REDIRECT SCHEME AFTER SIGN IN, e.g. myapp://]",
+                            "SignOutRedirectURI": "[CUSTOM REDIRECT SCHEME AFTER SIGN OUT, e.g. myapp://]",
+                            "Scopes": [ 
+                                "phone",
+                                "email",
+                                "openid",
+                                "profile",
+                                "aws.cognito.signin.user.admin"
+                            ]
+                        },
+                    }
+                }
+            }
+        }
+    }
+}''';
+```
+
+- **CredentialsProvider**:
+  - **Cognito Identity**:
+    - **Default**:
+      - **PoolID**: ID of the Amazon Cognito Identity Pool (e.g. `us-east-1:123e4567-e89b-12d3-a456-426614174000`)
+      - **Region**: AWS Region where the resources are provisioned (e.g. `us-east-1`)
+- **CognitoUserPool**:
+  - **Default**:
+    - **PoolId**: ID of the Amazon Cognito User Pool (e.g. `us-east-1_abcdefghi`)
+    - **AppClientId**: ID for the client used to authenticate against the user pool
+    - **Region**: AWS Region where the resources are provisioned (e.g. `us-east-1`)
+- **Auth**:
+  - **Default**:
+    - **authenticationFlowType**: The authentication flow type, takes values `USER_SRP_AUTH`, `CUSTOM_AUTH`. Default is `USER_SRP_AUTH`.
```

Comment:
```suggestion
    - **authenticationFlowType**: The authentication flow type, takes values `USER_SRP_AUTH`, `CUSTOM_AUTH`, and `USER_PASSWORD_AUTH`. Default is `USER_SRP_AUTH`.
```

Reply:
Updated in most recent commit. Glad you caught this so an option wasn't missing.